src/components/Card.jsx

```diff
@@ -0,0 +1,9 @@
+
+
+const Card = () => {
```

Comment:
Whats the need of this component?

Reply:
had created it to solve a bug, forgot to delete it later.